pkg/ccl/importccl/import_stmt.go

```diff
@@ -507,13 +507,13 @@ func importPlanHook(
 			var intoCols []string
 			var isTargetCol = make(map[string]bool)
 			for _, name := range importStmt.IntoCols {
-				var err error
-				if _, err = found.FindActiveColumnByName(name.String()); err != nil {
-					return errors.Wrap(err, "verifying target columns")
+				active, err := found.FindActiveColumnsByNames(tree.NameList{name})
+				if err != nil {
+					return errors.Wrapf(err, "verifying target columns: norm=%q str=%q", name.Normalize(), name.String())
```

Comment:
this error message might have been useful in debugging but from a user point of view is very cryptic. The expected case here is they specify something which isn't a column and the less noise we add to the "not found" error the better i think? i'd probably just leave it as it was.